sentry-samples/kmp-app/iosApp/iosApp.xcodeproj/project.pbxproj

```diff
@@ -165,6 +166,25 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		244CD82629E05BE700F8451F /* Senry dsyms upload */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}",
+			);
+			name = "Senry dsyms upload";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if which sentry-cli >/dev/null; then\nexport SENTRY_ORG=sentry-sdks\nexport SENTRY_PROJECT=sentry-kotlin-multiplatform\nexport SENTRY_AUTH_TOKEN=14e7bf400bc449589fdc32fad8beaad003456b05b784403fb11d109b6276cefe\nERROR=$(sentry-cli upload-dif \"$DWARF_DSYM_FOLDER_PATH\" 2>&1 >/dev/null)\nif [ ! $? -eq 0 ]; then\necho \"warning: sentry-cli - $ERROR\"\nfi\nelse\necho \"warning: sentry-cli not installed, download from https://github.com/getsentry/sentry-cli/releases\"\nfi\n";
```

Comment:
Please rotate the `SENTRY_AUTH_TOKEN` token and remove it from here, just add a place holder.

Reply:
weird that it shows up, I've specifically removed it. but yeah that shouldn't be there at all